.gitattributes

```diff
@@ -1,7 +1,73 @@
-# ensure CRLF endings on all checkouts
+###############################################################################
```

Comment:
The comments seems redundant and uninformative.
I find this one is more useful: https://gitattributes.io/

Reply:
Well, the comments are better than nothing:) The file is generated automatically by VS. If we don't edit it, it will be easier to update - just paste a new variant.

The output of gittattattributes.io is quite specific and looks less organized to me. Also, the output contains `diff=astextplain` which is msysgit only feature. (https://gitattributes.io/api/visualstudiocode%2Ccommon%2Ccsharp)

I would keep the current version for now and replace it if there will be any good looking suggestion.